monkey/infection_monkey/credential_collectors/ssh_collector/ssh_handler.py

```diff
@@ -1,7 +1,11 @@
 import glob
 import logging
 import os
-import pwd
+
+try:  # can't import on Windows
+    import pwd
```

Comment:
This is solved by Vakaris in parsing credential telemetry. Check [this commit](https://github.com/guardicore/monkey/commit/77b47f08b51f78601b98867395817dbd4266b20a)

Reply:
I'm going to leave this in since this'll most likely get merged before that.